.github/workflows/ci.yml

```diff
@@ -23,9 +25,33 @@ jobs:
     - uses: actions/setup-java@v1
       with:
         java-version: 11
-    - run: |
-        mvn --batch-mode package
+    - run: mvn --batch-mode deploy
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    # will be uncommented as soon as sonar token is available
+    # - run: mvn --batch-mode verify sonar:sonar -Dsonar.login=${SONAR_TOKEN}
+    #   env:
+    #     SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+    #     GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
```suggestion
```

Reply:
No otherwise sonarcloud will print error that github token is required.

Reply:
see blueprint
https://github.com/jhagestedt/blueprint/runs/703631738?check_suite_focus=true#step:6:502

Reply:
Yes, sorry. I thought plan to extend the step before. If you have an additional step for `sonar:sonar` omit the `verify` to save execution time. `verify` is implicitly executed in the step before.